### PR TITLE
add incident report 2018-01-04

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

**Comment by yuvipanda** [comment]
ty for the detailed report, @willingc! I wonder if we should open specific bugs for the action items (or link ones that exist)? Happy to merge this!

**Comment by willingc** [comment]
I think either one is fine (linking if it exists) or opening a new one if needed. Thanks.

File: docs/source/incident-reports/2018-01-04-failed-staging-deploy.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Process
+
+- Minimize tribal knowledge in the deployment process (i.e. all may not know that restarting a failed Travis build may clear the error.).
+- Simplify steps into a checklist especially ones that require human lookup of information or copy/paste.
+
+### mybinder.org-deploy
+
+- Minimize copy-paste errors by user
+- Add a "deployment to staging" checklist and a "deployment to prod" checklist to the documentation (possibly as files distinct from the `README`.)
```
Comment (choldgraf) [inline docs/source/incident-reports/2018-01-04-failed-staging-deploy.md]: I am a big fan of checklists, +100

**Comment by choldgraf** [comment]
Is there someplace that the Jupyter team has an incident report template? This is a nicely-organized report and I will probably just copy/paste from this one in the future, but just checking if such a template already exists :-)

**Comment by willingc** [comment]
Yes, we do have an incident report template [here](https://github.com/jupyterhub/mybinder.org-deploy/blob/staging/docs/source/incident-reports/template-incident-report.md). Probably makes sense to have a link from the README to it. @choldgraf

**Comment by choldgraf** [comment]
whoah! how did I miss that?! super cool, thanks for the link :-)

**Comment by minrk** [comment]
Excellent, thanks! Will start opening Issues.

**Pull request merged**